oam/install_scripts/mcs-loadbrm.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import configparser
+import subprocess
+
+config = configparser.ConfigParser()
+config.read('/etc/columnstore/storagemanager.cnf')
+
+storage = config['ObjectStorage']['service']
+region = config['S3']['region']
+bucket = config['S3']['bucket']
+loadbrm = '/usr/bin/load_brm /var/lib/columnstore/data1/systemFiles/dbrm/{0}'
+brm_saves_current = ''
+
+if storage.lower() == 's3' and not region.lower() == 'some_region' and not bucket.lower == 'some_bucket':
+    # load s3
+    brm = 'data1/systemFiles/dbrm/BRM_saves_current'
+
+    try:
+        brm_saves_current = subprocess.check_output(['smcat', brm])
+    except subprocess.CalledProcessError as e:
```

Comment:
Not doing anything w/ these exceptions because mcs-loadbrm will be logging any output to journalctl as it is